connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectIntegrationTestUtils.java

```diff
@@ -16,28 +16,26 @@
  */
 package org.apache.kafka.connect.integration;
 
-import org.junit.rules.TestRule;
-import org.junit.rules.TestWatcher;
-import org.junit.runner.Description;
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.TestWatcher;
 import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * A utility class for Connect's integration tests
  */
-public class ConnectIntegrationTestUtils {
-    public static TestRule newTestWatcher(Logger log) {
-        return new TestWatcher() {
-            @Override
-            protected void starting(Description description) {
-                super.starting(description);
-                log.info("Starting test {}", description.getMethodName());
-            }
+public class ConnectIntegrationTestUtils implements TestWatcher, BeforeEachCallback, AfterEachCallback {
```

Comment:
It seems this is used to log test name. Maybe we can move the log to each tests? There are only three class using this helper.

Reply:
It's a good idea, I will remove it